docs/intro.asciidoc

```diff
@@ -31,9 +31,9 @@ They make it easy to format your logs into ECS-compatible JSON. For example:
 Refer to the installation instructions of the individual loggers:
 
 * {ecs-logging-dotnet-ref}/setup.html[.NET]
-* Go: ({ecs-logging-go-zap-ref}/setup.html[zap]), ({ecs-logging-go-logrus-ref}/setup.html[logrus])
+* Go: {ecs-logging-go-zap-ref}/setup.html[zap], {ecs-logging-go-logrus-ref}/setup.html[logrus]
```

Comment:
Happy to be overridden here. I wasn't sure the value of the parens.

Reply:
There isn't one. Good fix 👍